### PR TITLE
Recommended mitigations section for Access Codes

### PR DESCRIPTION
**Pull request by mc2147**

### CODE REVIEW

File: docs/api-clients/access-codes/README.md [hunk 1]
```diff
@@ -101,6 +101,27 @@ Warnings are returned in a list:
...
+| `salto_site_user_not_subscribed`                   | Check the Salto user's subscription and your Salto site's subscription limit.                                                                                                                                                                                                                                                                                                                                                                              |
+| `smartthings_failed_to_set_after_multiple_retries` | Check for duplicate codes caused by third-party software managing the device. RBoy Apps in particular can cause this, so disable these for devices that are managed using Seam if possible. |
+| `smartthings_no_free_slots_available`              | Delete existing access codes on the device to free up slots for new access codes. Note that in addition to the codes managed by Seam, there are 3 other codes on the device.
```
Comment (seveibar) [inline docs/api-clients/access-codes/README.md:123]: this is good, `slots_full` should clearly be a generic error that covers a couple providers, but this is the best short term solution!

**Comment by seveibar** [comment]
@mc2147 I think we need a separate page for recommended mitigations, but this is a really good start and good to :ship: i think

**Review by seveibar — APPROVED** [review]

**Comment by seveibar** [comment]
Why a separate page? We made need to explain when to engage a backup code pool, recommended messaging to hosts and guests etc.

**Comment by mc2147** [comment]
> @mc2147 I think we need a separate page for recommended mitigations, but this is a really good start and good to 🚢 i think

yeah no worries I can move it to a separate page. Wasn't sure if we wanted it on the same page as the error codes for easier reference.

Do we just want to make a new page here then? All the other pages listed here are endpoint documentation just FYI:
image

**Comment by mc2147** [comment]
> Why a separate page? We may need to explain when to engage a backup code pool, recommended messaging to hosts and guests etc.

oh got it, makes sense

File: docs/api-clients/access-codes/README.md [hunk 2]
```diff
@@ -101,6 +101,27 @@ Warnings are returned in a list:
...
+| -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `failed_to_set_on_device`        | Check the error message for more specific mitigation steps or use a [backup access code](https://docs.seam.co/latest/core-concepts/access-codes#backup-access-codes)                                                                                                                                                               |
+| `failed_to_remove_from_device`   | Check the error message for more specific mitigation steps. Confirm using the provider's app if the access code is still on the device, as it may have been deleted through another source.                                                                                                                                                   |
```
Comment (rchodava) [inline docs/api-clients/access-codes/README.md:114]: hmm, what's the "it may have been deleted through another source" referring to?

Comment (mc2147) [reply docs/api-clients/access-codes/README.md:114]: For example if they deleted it through the provider's app, our deletion attempt might fail. I'm theorizing here but that's what I think happened a couple times with Kwikset

**Comment by mc2147** [comment]
I moved the mitigations to another page @seveibar @rchodava if you want to give it another look, otherwise I'll merge in a bit!

**Review by seveibar — APPROVED** [review]

**Pull request merged**